google/cloud/storage/doc/storage-main.dox

```diff
@@ -0,0 +1,97 @@
+/*!
+
+@mainpage Google Cloud Storage C++ Client Library
+
+These pages contain the reference guide to the Google Cloud Storage (GCS)
+C++ Client Library.
+
+## Quickstart
+
+The following instructions show you how to perform basic tasks in Cloud Storage
+using the C++ client library.
+
+Costs that you incur in Cloud Storage are based on the resources you use. This
+quickstart typically uses less than $0.01USD worth of Cloud Storage resources.
+
+If you have never used Cloud Storage we recommend that you first follow one of
+the [language neutral quickstart guides][quickstart-link] before you follow this
+guide.
+
+### Before you begin
+
+1. Select or create a Google Cloud Platform (GCP) project.
+
+[GO TO THE MANAGE RESOURCES PAGE][resource-link]
```

Comment:
These seem to terminate the numbered list (the following numbered items, despite starting with 2 and 3, all render as "1.").  Maybe these lines need to be indented, or there needs to be some special form of linebreak?  /shrug

Reply:
Fixed.